app/forms/user_piv_cac_verification_form.rb

```diff
@@ -21,6 +21,7 @@ def submit
   end
 
   def piv_cac_configuration
+    return nil if x509_dn_uuid.blank?
```

Comment:
Do we have test coverage for this already? (Can we?)

I'm assuming we're not expecting a meaningful change in behavior, but wondering if we'd previously accounted for an explicit `nil` return value.

Reply:
`::PivCacConfiguration.find_by(x509_dn_uuid: nil)` will return `nil`, so it should be safe to return nil directly.

Reply:
Looking a bit closer, it looks like the query in question is triggered by this: https://github.com/18F/identity-idp/blob/941d41a09aa607c8e90e08a80b6b9927beb0526d/app/controllers/two_factor_authentication/piv_cac_verification_controller.rb#L105 which is called even in the initial `#show` path which isn't trying to verify anything.

Reply:
I guess more what I was suggesting is to include a `context 'when x509_dn_uuid is nil' do` block in `spec/forms/user_piv_cac_verification_form_spec.rb` to prescribe the expected outcomes.